src/test/scala/org/ergoplatform/utils/generators/ChainGenerator.scala

```diff
@@ -68,11 +68,17 @@ trait ChainGenerator extends ErgoTestConstants {
   def popowHeaderChain(chain: HeaderChain): Seq[PoPowHeader] = {
     chain.headers.foldLeft((Seq.empty[PoPowHeader], None: Option[PoPowHeader])) {
       case ((acc, bestHeaderOpt), h) =>
-        val links = popowAlgos.updateInterlinks(
-          bestHeaderOpt.map(_.header),
-          bestHeaderOpt.map(ph => popowAlgos.interlinksToExtension(ph.interlinks).toExtension(ph.id))
-        )
-        val poPowH = PoPowHeader(h, links)
+        val links = if (bestHeaderOpt.isEmpty) {
+          Seq(scorex.util.bytesToId(Array.fill(32)(0: Byte)))
```

Comment:
Similarly here, the first iteration of this `foldLeft` requires a default interlink.